core/graphelier_news/requirements.txt

```diff
@@ -0,0 +1,10 @@
+astroid==2.3.3
+autopep8==1.5
+isort==4.3.21
+lazy-object-proxy==1.4.3
+mccabe==0.6.1
+nltk==3.4.5
+pycodestyle==2.5.0
+pylint==2.4.4
+six==1.14.0
+wrapt==1.11.2
```

Comment:
flask ?

Reply:
well I didn't use flask for the setup so i didn't see the need to install it now